### ISSUE TITLE
could not run react native app

### ISSUE THREAD

**Issue by deepthignair** [NativeScript/nativescript-cli#3739]
error: resource android:attr/colorError not found.
error: style attribute 'android:attr/keyboardNavigationCluster' not found.
error: failed linking references.

 Task :app:processDebugResources
Failed to execute aapt
com.android.ide.common.process.ProcessException: Failed to execute aapt

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugResources'.
> Failed to execute aapt

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 43s
15 actionable tasks: 1 executed, 14 up-to-date
Could not install the app on the device, read the error above for details.

Make sure you have an Android emulator running or a device connected and haveset up your Android development environment:
https://facebook.github.io/react-native/docs/getting-started.html

**Comment by rosen-vladimirov** [comment]
Hey @deepthignair ,
It looks like you've found NativeScript CLI's repository while trying to file an issue in react-native's one. 
In case you've not heard about NativeScript, I would like to recommend you to take a look at our [official documentation](https://docs.nativescript.org/core-concepts/technical-overview). You'll see that NativeScript's idea is really close to the React Native one, but there are some major differences. For example, in NativeScript it's really easy to call your native methods directly from JavaScript. You can find some of the differences described in [this blog post](https://www.nativescript.org/blog/would-airbnb-have-fared-better-with-nativescript-instead-of-react-native).
Regarding your current issue, I believe the best solution is to file a report in [react-native's repository](https://github.com/facebook/react-native). I may try to guess the problem is described in these lines:
```
error: resource android:attr/colorError not found.
error: style attribute 'android:attr/keyboardNavigationCluster' not found.
error: failed linking references.
```
Basically this means that some resources, that are referenced in your application (most probably in the AndroidManifest.xml) could not be found. You can check a similar issue [reported here](https://stackoverflow.com/questions/44643943/attr-colorerror-not-found-error-when-using-com-android-supportrecyclerview-v72)

But this is just my guess for the solution.

I'm closing this issue as it is not related to NativeScript.
